tests/test_build/test_sidebars_nested_page.html

```diff
@@ -13,23 +13,26 @@
     <a class="reference internal" href="index.html">
      Subsection 1.1 index
     </a>
-    <input checked="" class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox"/>
-    <label class="toctree-toggle" for="toctree-checkbox-1">
-     <i class="fa-solid fa-chevron-down">
-     </i>
-    </label>
-    <ul class="current">
-     <li class="toctree-l2 current active">
-      <a class="current reference internal" href="#">
-       Section 1 sub 1 page 1
-      </a>
-     </li>
-     <li class="toctree-l2">
-      <a class="reference internal" href="page2.html">
-       Section 1 sub 1 page 2
-      </a>
-     </li>
-    </ul>
+    <details open="">
```

Comment:
Note to reviewer: please double check me that this markup structure (here and in the other test file snapshots) matches the comments I added in toctree.py

Reply:
It seems to be in alignment, though I left a comment on the boolean syntax for consideration.